cmlreaders/cmlreader.py

```diff
@@ -72,4 +91,67 @@ def load(self, data_type, file_path=None):
                                        localization=self.localization,
                                        montage=self.montage,
                                        file_path=file_path,
-                                       rootdir=self.rootdir).load()
+                                       rootdir=self.rootdir).load(**kwargs)
+
+    def load_eeg(self, events: Optional[pd.DataFrame] = None,
+                 pre: int = 0, post: int = 0,
+                 epochs: Optional[List[Tuple[int, int]]] = None,
+                 contacts: Optional[pd.DataFrame] = None,
+                 scheme: Optional[pd.DataFrame] = None):
+        """Load EEG data.
+
+        Keyword arguments
+        -----------------
+        events
+            Events to load EEG epochs from. Incompatible with passing
+            ``epochs``.
+        pre
+            Time in ms to include prior to passed events (default: 0). This
+            parameter has no effect if events are not specified.
+        post
+            Time in ms to include after passed events (default: 0). This
+            parameter has no effect if events are not specified.
+        epochs
+            A list of ``(start, stop)`` tuples to specify epochs to retrieve
+            data from. Incompatible with passing ``events``.
+        contacts
```

Comment:
Since this is being passed as a dataframe, we will internally need to know which column to use for the channel label. Would it make sense to add another keyword argument where the user can specify the column to use? Or, should it just assume the default column name based on what is returned when you load contacts?

Reply:
If I decide I want to use the "epochs" parameter to define the time periods of interest, and my epochs aren't all the same length, what should happen?